example/ipsec_api/odp_ipsec_run_ah_in

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# Test input AH
+#  - 2 loop interfaces
+#  - 10 packets
+#  - Specify API mode on command line
+odp_ipsec -i loop1,loop2 \
+-r 192.168.111.2/32:loop1:08.00.27.76.B5.E0 \
+-p 192.168.222.0/24:192.168.111.0/24:in:ah \
+-a 192.168.222.2:192.168.111.2:md5:300:27f6d123d7077b361662fc6e451f65d8 \
+-s 192.168.222.2:192.168.111.2:loop2:loop1:10:100 \
+-c 2 "$@"
```

Comment:
Checkpatch seems confused here. Is it expecting a .sh suffix? Any thoughts on this @muvarov?
```
ERROR: do not set execute permissions for source files
#1770: FILE: example/ipsec_api/odp_ipsec_run_ah_in
```
The rest of the shell files are similarly flagged.

Reply:
@Bill-Fischofer-Linaro yes, it expects .sh files. I can change filenames, if you wish.

Reply:
We use that convention elsewhere (_e.g.,_ in the various `test/performance` scripts) so I think that's a good convention to follow.